pywbem/cim_operations.py

```diff
@@ -1220,68 +1223,74 @@ def configure_logger(cls, name, log_dest=DEFAULT_LOG_DESTINATION,
+            Simple name (ex. `'api'`) of the single pywbem logger this method
+            should affect, or `'all'` to affect all pywbem loggers.
+
+            Must be one of the strings in
+            :data:`~pywbem._logging.LOGGER_SIMPLE_NAMES`.
 
           log_dest (:term:`string`):
-            String defining the destination for this log or None.
+            Log destination for the affected pywbem loggers, controlling the
+            configuration of its Python logging parameters (log handler,
+            message format, and log level).
 
-            If it is a :term:`string` defining one of the destinations defined
-            in :data:`~pywbem._logging.LOG_DESTINATIONS` a Python logger is
-            configured for that destination including configuring a logging
-            handler, the message format and setting the log level to DEBUG.
+            If it is a :term:`string`, it must be one of the strings in
+            :data:`~pywbem._logging.LOG_DESTINATIONS` and the Python logging
+            parameters of the loggers will be configured accordingly for their
+            log handler, message format, and with a logging level of
+            :attr:`py:logging.DEBUG`.
 
-            If `None` a logger with the Python Python NullHandler is configured.
+            If `None`, the Python logging parameters of the loggers will not be
+            changed.
 
-          detail_level (:term:`string`, :class:`int` or None):
-            Defines the level of detail contained in each log record created.
+          detail_level (:term:`string` or :class:`int` or `None`):
+            Detail level for the data in each log record that is generated by
+            the affected pywbem loggers.
 
-            If if is a :term:`string` it must be one of the key words that
-            defines the level of detail to be output defined in
-            :data:`~pywbem._logging.LOG_DETAIL_LEVELS`.
+            If it is a :term:`string`, it must be one of the strings in
+            :data:`~pywbem._logging.LOG_DETAIL_LEVELS` and the loggers will
+            be configured for the corresponding detail level.
 
-            If  it is a :class: `int` it defines the maximum size of the log
-            record record created.
+            If it is an :class:`int`, it defines the maximum size of the log
+            records created and the loggers will be configured to output all
+            available information up to that size.
 
-            If `None` it is set to the default detail level for that logger.
+            If `None`, the detail level configuration will not be changed.
 
           log_filename (:term:`string`):
-            Filename to use as logging file if the log destination is `file`.
-            Ignored if log destination is not `file`. This parameter is
-            required if the dest parameter is `file`.
+            Path name of the log file (required if the log destination is
```

Comment:
Not required.  That was my error. See pr for issue # 1167 that states that there is a default log file name.

Reply:
**Discussion point:** If we have a default log file name, we should do that for both configure functions, and we should not use the same constant for the path name of the file that we use for the log destination.